CMakeLists.txt

```diff
@@ -42,9 +43,6 @@ include(logging)
 # include exernal dependencies
 include(dependencies)
 
-include(${PROJECT_SOURCE_DIR}/build/conanbuildinfo.cmake)
-conan_basic_setup(KEEP_RPATHS)
```

Comment:
Do we still need this? I can't remember what problem it was fixing?

Reply:
I feel like this was a Mac thing